spec/features/agents/rdv_details_spec.rb

```diff
@@ -7,16 +7,18 @@
     let(:organisation) { create(:organisation) }
     let(:service) { create(:service) }
     let(:motif) { create(:motif, service: service, name: "Renseignements") }
-    let!(:rdv) { create(:rdv, agents: [agent], motif: motif, organisation: organisation, starts_at: Time.zone.local(2022, 4, 7).at_noon) }
+    let!(:rdv) { create(:rdv, agents: [agent], motif: motif, organisation: organisation, starts_at: starts_at) }
+    let(:starts_at) { Time.zone.local(2022, 4, 7).at_noon }
     let!(:receipt) { create(:receipt, rdv: rdv, result: :sent, content: "Vous avez rendez-vous!") }
     let(:agent) { create(:agent, first_name: "Bruce", last_name: "Wayne", service: service, basic_role_in_organisations: [organisation]) }
 
     before do
       login_as(agent, scope: :agent)
-      visit admin_organisation_rdvs_path(organisation)
     end
 
     it do
```

Comment:
```suggestion
    it "allows listing RDVs" do
```

Et au passage, le `describe "See RDV details" do` ligne 6 semble superflu. :wink: